cmd/fortifyExecuteScan_test.go

```diff
@@ -610,3 +610,37 @@ func TestAutoresolveClasspath(t *testing.T) {
 		assert.Equal(t, "some.jar;someother.jar", result, "Expected different result")
 	})
 }
+
+func TestPopulateMavenTranslate(t *testing.T) {
+	t.Run("src without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Src: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"classpath":"","src":"./**/*"}]`, translate, "Expected different parameters")
+	})
+
+	t.Run("exclude without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Exclude: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"classpath":"","exclude":"./**/*"}]`, translate, "Expected different parameters")
+	})
+
```

Comment:
I would add one more test-case which confirms that `config.Translate` is used verbatim if provided and `config.Src` and `config.Exclude` are both ignored.